website/source/docs/configuration/storage/etcd.html.md

```diff
@@ -122,3 +138,4 @@ storage "etcd" {
 ```
 
 [etcd]: https://coreos.com/etcd "Etcd by CoreOS"
+[dns discovery]: https://coreos.com/etcd/docs/latest/v2/clustering.html#dns-discovery "Etcd cluster DNS Discovery"
```

Comment:
use this link instead: https://coreos.com/etcd/docs/latest/op-guide/clustering.html#dns-discovery